SteamKit2/SteamKit2/Steam/Handlers/SteamApps/SteamApps.cs

```diff
@@ -263,7 +239,7 @@ public AsyncJobMultiple<PICSProductInfoCallback> PICSGetProductInfo( IEnumerable
                 var appinfo = new CMsgClientPICSProductInfoRequest.AppInfo();
                 appinfo.access_token = app_request.AccessToken;
                 appinfo.appid = app_request.ID;
-                appinfo.only_public_obsolete = app_request.Public;
+                appinfo.only_public_obsolete = false;
```

Comment:
can we just omit it entirely here?

Reply:
We can, but I don't know what Valve defaults to on the backend, if at all (since it doesn't seem to have an effect).